src/OpenTelemetry.Instrumentation.GrpcCore/ServerTracingInterceptorOptions.cs

```diff
@@ -27,7 +27,7 @@ public class ServerTracingInterceptorOptions
     public TextMapPropagator Propagator { get; internal set; } = Propagators.DefaultTextMapPropagator;
 
     /// <summary>
-    /// Gets or sets a custom identfier used during unit testing.
+    /// Gets or sets custom activity tags used during unit testing.
     /// </summary>
-    internal Guid ActivityIdentifierValue { get; set; }
+    internal IReadOnlyDictionary<string, object> ActivityTags { get; set; }
```

Comment:
Please cosnider - TestingsTags/AdditionalTags.

Reply:
Renamed to `AdditionalTags`, looks good.